grafana-plugin/src/containers/PluginConfigPage/PluginConfigPage.tsx

```diff
@@ -134,23 +129,30 @@ export const PluginConfigPage = (props: Props) => {
     setGrafanaUrl(e.target.value);
   }, []);
 
-  const handleSyncException = useCallback((e) => {
-    const buildErrMsg = (msg: string): string =>
-      constructSyncErrorMessage(msg, plugin.meta.jsonData?.onCallApiUrl);
+  const handleSyncException = useCallback((e: AxiosError) => {
+    const buildErrMsg = (msg: string): string => constructSyncErrorMessage(msg, plugin.meta.jsonData?.onCallApiUrl);
 
     if (plugin.meta.jsonData?.onCallApiUrl) {
-      const { status: statusCode  } = e.response;
+      const { status: statusCode } = e.response;
 
       let statusMessage: string;
 
-      if (statusCode == 403) {
+      if (statusCode === 403) {
```

Comment:
@maskin25 here as well this should be a safe change. the `e` arg to this anonymous function represents an `AxiosError`, I've updated the typing to reflect this